script/github-actions/filter-files-changed.js

```diff
@@ -1,5 +1,10 @@
+/* eslint-disable no-console */
 const args = process.argv.slice(2);
 const files = args[0].slice(1, -1).split(','); // remove unnecessary characters
-const filteredFiles = files.filter(file => /.+\.jsx?$/.test(file)).join(' ');
+const filteredJSFiles = files.filter(file => /.+\.jsx?$/.test(file)).join(' ');
+const filteredSCSSFiles = files
+  .filter(file => /.+\.scss?/.test(file))
```

Comment:
Should the regex here be `.+\.s?css$`? Not sure if `stylelint` is meant to go through `.css` files too. If not, then just `.+\.scss$` would work.

Reply:
In package.json, `lint:sass` runs `stylelint src/**/*.scss` and that is what I have been testing against. @bkjohnson maybe you can verify if we also need css included?

Reply:
If we have CSS files, then `stylelint` should go through them. I made the (possibly incorrect) assumption that we _only_ had `.scss` files, and modeled the configuration after `sass-lint`:

https://github.com/department-of-veterans-affairs/vets-website/blob/3e58cf9956aedf964d9455c18c8e8c3038be77bc/config/sass-lint.yml#L16-L17